### PR TITLE
Unspecified changes by the management committee in the near future

### PR DESCRIPTION
**Pull request by bradleysigma**
Changes that don't affect the meaning of the constitution, for example
* renumbering sections that became misnumbered due to the numerous simultaneous pull requests
* minor and obvious spelling/grammar mistakes
* reverting any changes not approved by the Union

### CODE REVIEW

**Comment by kumar-ish** [comment]
Would this change be incorporated into the constitution? If yes, then make the changes that the management committee can do to the constitution an exhaustive list (instead of listing a few examples).

**Comment by jdcaperon** [comment]
I think this PR can be closed as the management committee reserves the right to formatting the rules.
Fixing grammar/spelling mistakes however may change the interpretation of rules and should be passed at general meetings in my opinion.

**Comment by bradleysigma** [comment]
Approved at AGM 8th Oct 2019.

File: constitution.md [hunk 1]
```diff
@@ -1,87 +1,76 @@
...
+5.1 A member may resign from the Society at any time by giving notice in writing to the Secretary.
 
-6.3 If a member or applicant for membership:
+5.2 Such resignation shall take effect at the time such notice is received by the Secretary, unless a later date is specified in the notice, when it shall take effect on the later date.
 
+5.3 If a member or applicant for membership:  
 * fails to comply with any of the provisions of the Constitution;  
 * has membership fees in arrears;  
 * violates the Society's Code of Conduct; or  
 * acts in a manner considered to be injurious or prejudicial to the character or interests of the Society
```
Comment (kumar-ish) [inline constitution.md]: Shouldn't there be a comma at the end of this line?

File: constitution.md [hunk 2]
```diff
@@ -168,14 +157,11 @@ the membership shall consider at a General Meeting whether that person's members
 
 ## 11 Functions of the Management Committee
 
-11.1 Except as otherwise provided by these rules and subject to resolutions of the members of the Society carried at any general meeting the management committee:
-
-* shall have the general control and management of the administration of the affairs and funds of the Society
-
+11.1 Except as otherwise provided by these rules and subject to resolutions of the members of the Society carried at any general meeting the management committee:
```
Comment (kumar-ish) [inline constitution.md]: ...at any general meeting**,** ...

**Comment by kumar-ish** [comment]
Unless I am misunderstanding this, "The management committee will take full responsibility for all publications produced by the Society or by any of its members" should be reworded to something along the lines of "The management committee will take full responsibility for all publications produced by its members or by the Society" to ensure that 'its members' doesn't refer to the members of the Society, but the management committee.

**Comment by jdcaperon** [comment]
The latter change would have to observe section 21 as it changes the definition of the rule.

**Comment by kumar-ish** [comment]
Should I be good to merge this now?

**Comment by jdcaperon** [comment]
@bradleysigma @kumar-ish @nicklambourne can we close or merge this PR?

**Comment by bradleysigma** [comment]
Well, the numbering in `master` is broken, so we can't close it, and the numbering in this branch is broken, so we can't merge it. It's now @sannibunny's problem.

**Comment by kumar-ish** [comment]
If new T3 can force push the numbering fixes to this branch then we can merge it @sannibunny @kentonlam @JamesDearlove 👍

**Comment by sannibunny** [comment]
Just to clarify, which numbers are messed up in this PR? They look correct to me under "Files Changed" but I could be wrong...

**Comment by kumar-ish** [comment]
8.7 is repeated. I think there was another error but I can't find it 🤔

**Comment by sannibunny** [comment]
Fixed the numbering in this branch, I am merging this PR now.

**Pull request merged**